### DISCUSSION TITLE
Cannot create property 'url' on boolean 'true'

### DISCUSSION THREAD

**Discussion by FalkWinkler** [Koenkk/zigbee2mqtt#19706] [Show and tell]
If i  use the Environment Variables 
`ZIGBEE2MQTT_CONFIG_FRONTEND_PORT` or `ZIGBEE2MQTT_CONFIG_FRONTEND_URL` 
i have configuration Errors

I deploy in kubenetes:

`apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: zigbee2mqtt
spec:
  selector:
    matchLabels:
      app: zigbee2mqtt
  serviceName: zigbee2mqtt
  template:
    metadata:
      labels:
        app: zigbee2mqtt
    spec:
      containers:
      - name: zigbee2mqtt
        image: ghcr.io/koenkk/zigbee2mqtt:latest
        resources:
          limits:
            memory: "128Mi"
            cpu: "500m"
        ports:
        - containerPort: 8080
        volumeMounts:
          - name: data
            mountPath: /data

        env:
          - name: ZIGBEE2MQTT_CONFIG_MQTT_BASE_TOPIC
            value: "zigbee2mqtt"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_INCLUDE_DEVICE_INFORMATION
            value: "true"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_KEEPALIVE
            value: "60"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_REJECT_UNAUTHORIZED
            value: "true"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_SERVER
            value: "mqtt://mosquitto.home-automation.svc.cluster.local"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_VERSION
            value: "5"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_USER
            value: "admin"
          - name: ZIGBEE2MQTT_CONFIG_MQTT_PASSWORD
            value: admin@123
          - name: ZIGBEE2MQTT_CONFIG_PERMIT_JOIN
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_SERIAL_PORT
            value: "tcp://ser2sock.home-automation.svc.cluster.local:10000"
          - name: ZIGBEE2MQTT_CONFIG_SERIAL_BAUDRATE
            value: "115200"
          - name: ZIGBEE2MQTT_CONFIG_SERIAL_DISABLE_LED
            value: "false"
          - name: ZIGBEE2MQTT_DATA
            value: /data
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_HOMEASSISTANT_DISCOVERY_TOPIC
            value: homeassistant
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_HOMEASSISTANT_LEGACY_ENTITY_ATTRIBUTES
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_HOMEASSISTANT_LEGACY_TRIGGERS
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_HOMEASSISTANT_STATUS_TOPIC
            value: homeassistant/status
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_LAST_SEEN
            value: ISO_8601
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_LEGACY_API
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_LEGACY_AVAILABILITY_PAYLOAD
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_LOG_LEVEL
            value: info
          - name: ZIGBEE2MQTT_CONFIG_ADVANCED_LOG_OUTPUT
            value: '["console"]'
          - name: ZIGBEE2MQTT_CONFIG_AVAILABILITY_ACTIVE_TIMEOUT
            value: "60"
          - name: ZIGBEE2MQTT_CONFIG_AVAILABILITY_PASSIVE_TIMEOUT
            value: "2000"
          - name: ZIGBEE2MQTT_CONFIG_DEVICE_OPTIONS_LEGACY
            value: "false"
          - name: ZIGBEE2MQTT_CONFIG_DEVICE_OPTIONS_RETAIN
            value: "true"
          - name: ZIGBEE2MQTT_CONFIG_EXPERIMENTAL_NEW_API
            value: "true"
          - name: ZIGBEE2MQTT_CONFIG_HOMEASSISTANT
            value: "true"
          # - name: ZIGBEE2MQTT_CONFIG_FRONTEND_PORT
          #   value: "8080"
          - name: ZIGBEE2MQTT_CONFIG_FRONTEND_URL
            value: "https://zigbee2mqtt.test"

  volumeClaimTemplates:
  - metadata:
      name: data
    spec:
      accessModes: [ "ReadWriteOnce" ]
      storageClassName: ceph-block
      resources:
        requests:
          storage: 1Gi`

without the FRONTEND properties works fine.

**Comment by jlejeune** [comment]
Same issue on my side.
Any news to solve it ?

**Reply by rwlove** [reply]
I have this issue too.
